sqlite/src/sqliteInt.h

```diff
@@ -4382,6 +4382,8 @@ u64 sqlite3LogEstToInt(LogEst);
 VList *sqlite3VListAdd(sqlite3*,VList*,const char*,int,int);
 const char *sqlite3VListNumToName(VList*,int);
 int sqlite3VListNameToNum(VList*,const char*,int);
+VList *sqlite3VListClone(const VList*, void* (*)(size_t));
+void sqlite3VListPrint(loglvl lvl, const VList *pIn);
 
```

Comment:
Ideally, these new prototypes should be wrapped in the #if SQLITE_BUILDING_FOR_COMDB2 found elsewhere in the file.

Reply:
done